pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -201,7 +201,22 @@ class CollectionViewSet(
     filterset_class = CollectionFilter
     pagination_class = LimitOffsetPagination
 
-    DEFAULT_ACCESS_POLICY = _PERMISSIVE_ACCESS_POLICY
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": "destroy",
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_or_obj_perms:ansible.delete_collection",
```

Comment:
This should just be model permissions. This action will remove the collection from every repository in the system which is more of an admin level function. (Also, this is the exception to the "don't use collection permissions" rule)